modules/ROOT/pages/salesforce/salesforce-mktg-connector-upgrade-migrate.adoc

```diff
@@ -0,0 +1,112 @@
+= Salesforce Marketing Cloud - Upgrade and Migrate - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../../assets/images/
+
+Upgrade Anypoint Connector for Salesforce Marketing Cloud (Salesforce Marketing Cloud Connector) to version 3.0.0.
```

Comment:
This seems kind of terse to me; is this templated? Can we offer more here, such as WHY a user wants or needs to do this? Or is it obvious?

Reply:
This is templated. We can revisit when we update the template for this topic.